pkg/apis/management.cattle.io/v3/machine_types.go

```diff
@@ -269,28 +269,46 @@ type NodeCommonParams struct {
 }
 
 // +genclient
-// +kubebuilder:skipversion
 // +genclient:nonNamespaced
+// +kubebuilder:object:root=true
+// +kubebuilder:resource:scope=Cluster
+// +kubebuilder:subresource:status
+// +kubebuilder:printcolumn:name="Activated",type=string,JSONPath=".spec.active"
+// +kubebuilder:printcolumn:name="Binary URL",type=string,JSONPath=".spec.url"
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
+// NodeDriver represents a Rancher node driver for a specific cloud provider used to provision cluster nodes.
 type NodeDriver struct {
 	metav1.TypeMeta `json:",inline"`
 	// Standard object’s metadata. More info:
 	// https://github.com/kubernetes/community/blob/master/contributors/devel/api-conventions.md#metadata
 	metav1.ObjectMeta `json:"metadata,omitempty"`
-	// Specification of the desired behavior of the the cluster. More info:
+	// Specification of the desired behavior of the cluster. More info:
```

Comment:
of the what now

Reply:
lol good point, noticed `the the` but didn't realize the rest was just copy pasta. Searching for that string it looks like this is present on a lot of other types too, like `ClusterRegistrationToken`, and `DynamicSchema`. Something to keep an eye out for when migrating other types in the future